.nvmrc

```diff
@@ -0,0 +1 @@
+v18.20.4
```

Comment:
Is this file necessary?

Reply:
Will replace with an env var in the platform